frontend/pages/organizations/[organizationUrl].js

```diff
@@ -205,9 +210,15 @@ function OrganizationLayout({
           <Typography color="primary" className={classes.headline} component="h2">
             {texts.this_organizations_projects}
           </Typography>
-          <Button variant="contained" color="primary" href={getLocalePrefix(locale) + "/share"}>
-            {texts.share_a_project}
-          </Button>
+          <Link
+            href={getLocalePrefix(locale) + "/share"}
+          >
+            <ControlPointSharpIcon
+              className={classes.button}
+              variant="contained"
+              color="primary"
+            />
+          </Link>
```

Comment:
I'd use the `<IconButton>` component instead of link here - similar to the share button. This will add some interactivity on hover and click. Also I'd only display the plus icon on mobile On larger screens the "Share a project" button is more descriptive. 
```suggestion
          {!isTinyScreen && (
            <Button variant="contained" color="primary" href={getLocalePrefix(locale) + "/share"}>
              {texts.share_a_project}
            </Button>
          ) : (
            <IconButton
              href={getLocalePrefix(locale) + "/share"}
            >
              <ControlPointSharpIcon
                className={classes.button}
                variant="contained"
                color="primary"
              />
            </IconButton>
          )
          }
```